release-schema.json

```
@@ -196,6 +196,48 @@
           "minLength": 1
         }
       }
+    },
+    "ExclusionGrounds": {
+      "properties": {
+        "sources": {
+          "title": "Sources",
+          "description": "The sources where the exclusion grounds are defined.",
```

eForms uses "where" (without a noun to modify), but once we add this noun, "in which" is better.

```suggestion
          "description": "The sources in which the exclusion grounds are defined.",
```